<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OpenLibraryDialog</class>
 <widget class="QDialog" name="OpenLibraryDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>516</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Open library</string>
  </property>
  <widget class="QPushButton" name="closeButton">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>260</y>
     <width>95</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QLabel" name="descriptionLibraryText">
   <property name="geometry">
    <rect>
     <x>282</x>
     <y>13</y>
     <width>221</width>
     <height>181</height>
    </rect>
   </property>
   <property name="text">
    <string>Select library for description</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QPushButton" name="choiceButton">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>260</y>
     <width>90</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Select</string>
   </property>
  </widget>
  <widget class="QListView" name="listLibrarys">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>231</width>
     <height>241</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="removeButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>260</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Remove</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>149</x>
     <y>260</y>
     <width>91</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
